src/types/systemIntake.ts

```diff
@@ -53,8 +53,9 @@ export type SystemIntakeState = {
 // Redux store type for systems
 export type SystemIntakesState = {
   systemIntakes: SystemIntakeForm[];
-  isLoading?: boolean;
-  loadedTimestamp?: DateTime;
+  isLoading: boolean | null;
```

Comment:
I prefer the `initialState` to contain all keys of state. If it isn't included, it would be `undefined` rather than `null`.

It would work either way; I just like to see that the key is there, rather than undefined.

One case where I would prefer something be `undefined` is in form data, because `null` would cause issues for controlled components.